dist/verbalexpressions.js

```diff
@@ -1,473 +1,601 @@
+         * @param {String} source string to search for
+         * @param {String} value value to replace with
+         * @return {VerbalExpression} same instace of VerbalExpression
+         */
+
+    }, {
+        key: 'replace',
+        value: function replace(source, value) {
+            source = source.toString();
+            return source.replace(this, value);
         }
-    };
 
-    /**
-    * @return {VerbalExpression} Returns a new instance of VerbalExpressions
-    */
-    function createVerbalExpression() {
-        return new VerbalExpression();
-    }
+        /**
+         * Convert to RegExp object
+         * @return {RegExp} Converted RegExp instance
+         */
+
+    }, {
+        key: 'toRegExp',
+        value: function toRegExp() {
+            var components = this.toString().match(/\/(.*)\/([gimuy]+)?/);
+            var pattern = components[1];
+            var flags = components[2];
+
+            return new RegExp(pattern, flags);
+        }
+    }], [{
+        key: 'sanitize',
+        value: function sanitize(value) {
+            if (value.source) {
+                return value.source;
+            }
+
+            if (typeof value === 'number') {
+                return value;
+            }
+
+            // Regular expression to match meta characters
+            // https://developer.mozilla.org/en/JavaScript/Reference/Global_Objects/regexp
+            var toEscape = /([\].|*?+(){}^$\\:=[])/g;
+
+            // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/lastMatch
+            var lastMatch = '$&';
+
+            // Escape meta characters
+            return value.replace(toEscape, '\\' + lastMatch);
+        }
+    }]);
 
-    // UMD (Universal Module Definition), URL: https://github.com/umdjs/umd
-    // Supports AMD, CommonJS and the browser
-    if (typeof module !== 'undefined' && module.exports) {
-        // Node.js Module
-        module.exports = createVerbalExpression;
-    } else if (typeof define === 'function' && define.amd) {
-        // AMD Module
-        define(MODULE_NAME, [], function define() {
-            return VerbalExpression;
-        });
+    return VerbalExpression;
+}(_extendableBuiltin(RegExp));
+
+// UMD (Universal Module Definition)
+// https://github.com/umdjs/umd
+
+
+(function (root, factory) {
+    if (typeof define === 'function' && define.amd) {
+        // AMD. Register as an anonymous module.
+        define([], factory);
+    } else if ((typeof module === 'undefined' ? 'undefined' : _typeof(module)) === 'object' && module.exports) {
+        // Node. Does not work with strict CommonJS
+        module.exports = factory();
     } else {
-        // Browser
-        root[MODULE_NAME] = createVerbalExpression;
+        // Browser globals (root is window)
+        root.VerEx = factory();
     }
-}(this));
+})(undefined, function () {
```

Comment:
Now as you see, `undefined` is being passed to the UMD function, which causes the script to fail. You should be able to pass the global `this` as a property for the UMD function.

Reply:
Yeah. Looks like that has been fixed now. Had to pass an option to babel.